ox_bqpipeline/bqpipeline.py

```diff
@@ -324,8 +440,13 @@ def run_query(self, path, batch=False, wait=True, create=True,
         template = self.jinja2.from_string(template_str)
         query = template.render(**kwargs)
         client = self.get_client()
+        job_config = self.create_job_config(batch, dest, create, overwrite)
+        query_parameters = self.set_query_params(query_params)
```

Comment:
nit avoid unnecessary `self.set_query_params` call when `query_params is None` which is true by default.
Also can simplify to 
```python
       if query_params:
           job_config.query_parameters = self.set_query_params(query_params)
```

Reply:
Done.